### PR TITLE
Gui test for Link storage by URL

### PR DESCRIPTION
**Pull request by maryvictol**
This PR add test for "Link storage with their URLs and not only with their IDs"

### CODE REVIEW

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/LaunchLimitMountsTest.java [hunk 1]
```diff
@@ -22,8 +22,15 @@
...
 import com.epam.pipeline.autotests.utils.C;
+import com.epam.pipeline.autotests.utils.ConfigurationProfile;
+import static com.epam.pipeline.autotests.utils.Json.selectProfileWithName;
+import static com.epam.pipeline.autotests.utils.Json.transferringJsonToObject;
+import com.epam.pipeline.autotests.utils.Parameter;
 import com.epam.pipeline.autotests.utils.TestCase;
 import com.epam.pipeline.autotests.utils.Utils;
+import static com.epam.pipeline.autotests.utils.Utils.removeStorages;
+import com.sun.webkit.network.URLs;
+import static org.testng.Assert.assertNotEquals;
```
Comment (kamyshova) [inline e2e/gui/src/test/java/com/epam/pipeline/autotests/LaunchLimitMountsTest.java]: Clean up the unused import statements, please

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/LaunchLimitMountsTest.java [hunk 2]
```diff
@@ -384,6 +390,46 @@ public void allowToLaunchRunsWithoutMounts() {
                         .close());
     }
 
+    @Test(priority = 4)
+    @TestCase(value = {"TC-PARAMETERS-5"})
+    public void linkStorageWithTheirURLsAndNotOnlyWithTheirIDs() {
+        library()
+                .createStorage(storage5)
+                .selectStorage(storage5)
+                .createFileWithContent("test_file1.txt", "test file1");
```
Comment (kamyshova) [inline e2e/gui/src/test/java/com/epam/pipeline/autotests/LaunchLimitMountsTest.java]: Let's extract strings to variables

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/LaunchLimitMountsTest.java [hunk 3]
```diff
@@ -384,6 +390,46 @@ public void allowToLaunchRunsWithoutMounts() {
...
+        };
+
+        library()
+                .createPipeline(pipeline1)
+                .clickOnDraftVersion(pipeline1)
+                .codeTab()
+                .clickOnFile("config.json")
+                .editFile(transferringJsonToObject(profiles -> {
+                    final ConfigurationProfile profile = selectProfileWithName("default", profiles);
+                    profile.configuration.parameters.put("CP_CAP_LIMIT_MOUNTS",
```
Comment (kamyshova) [inline e2e/gui/src/test/java/com/epam/pipeline/autotests/LaunchLimitMountsTest.java]: Let's extract "CP_CAP_LIMIT_MOUNTS" to constant and replace its everywhere

**Review by kamyshova — CHANGES_REQUESTED** [review]

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/LaunchLimitMountsTest.java (hunk 1 above)
Comment (maryvictol) [reply e2e/gui/src/test/java/com/epam/pipeline/autotests/LaunchLimitMountsTest.java]: Done

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/LaunchLimitMountsTest.java (hunk 2 above)
Comment (maryvictol) [reply e2e/gui/src/test/java/com/epam/pipeline/autotests/LaunchLimitMountsTest.java]: Done

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/LaunchLimitMountsTest.java (hunk 3 above)
Comment (maryvictol) [reply e2e/gui/src/test/java/com/epam/pipeline/autotests/LaunchLimitMountsTest.java]: Done

File: docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/TC-PARAMETERS-5.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+| Steps | Actions | Expected results |
+|:---:|---|---|
+| 1 | Open the **Library** page | |
+| 2 | Open the pipeline created at step 3 of the preparations | |
+| 3 | Click the pipeline draft version | |
+| 4 | Click the **CODE** tab | |
+| 5 | Click the `config.json` file | |
+| 6 | Click the **EDIT** button | |
+| 7 | Insert the following code into the `parameters` section: <br> `"CP_CAP_LIMIT_MOUNTS" : {` <br> `"value" : "<test_storage1_id>,<test_storage2_name>",` <br> `"type" : "string",` <br> `"required" : false` <br> `}` <br> where `<test_storage1_id>` - `test_storage1` id ,`<test_storage2_name>` - `test_storage1` name | |
```
Comment (NShaforostov) [inline docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/TC-PARAMETERS-5.md]: Fix to: "... where `<test_storage1_id>` - `test_storage1` id ,`<test_storage2_name>` - `test_storage2` name..."

**Review by NShaforostov — APPROVED** [review]
Fix a minor note before the merge please.

**Review by kamyshova — APPROVED** [review]

**Pull request merged**